feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -0,0 +1,221 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { Product, ProductScope, ProductService } from '@spartacus/core';
+import { BehaviorSubject, combineLatest, Observable } from 'rxjs';
+import { map, tap } from 'rxjs/operators';
+import { Configurator } from '../../../core/model/configurator.model';
+import { QuantityUpdateEvent } from '../../form/configurator-form.event';
+import { ConfiguratorPriceComponentOptions } from '../../price/configurator-price.component';
+import {
+  ConfiguratorAttributeQuantityComponentOptions,
+  Quantity,
+} from '../quantity/configurator-attribute-quantity.component';
+
+export interface ProductExtended extends Product {
```

Comment:
Can we move this `ProductExtended` out of this component? I see `ProductExtended` also defined in `ConfiguratorOverviewBundleAttributeComponent`. BTW, we can augment property to core model in library. So, `noLink` can be augmented to `Product` in this library.

Reply:
picked

Reply:
But we should not augment this for the entire library. This is a quite component specific enhancement

Reply:
The ProductExtended interface was obsolete. I have removed it as well as related program logic.